src/main.py

```diff
@@ -94,6 +98,7 @@
 #  there as well.
 
 _CONFIG = {
+    'template': base.constants.JINJA2,
```

Comment:
Oops, I updated the comment above but forgot to change the actual default value :-)